apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -218,9 +220,9 @@ internal string DatabaseToDrop
         public static SpannerCommandTextBuilder FromCommandText(string commandText)
         {
             GaxPreconditions.CheckNotNullOrEmpty(commandText, nameof(commandText));
-            commandText = commandText.Trim();
+            var trimmedCommandText = RemoveCommentsAndStatementHint(commandText).Trim();
```

Comment:
`commandText` is still being used in many places, let's keep it trimmed.
```suggestion
            var trimmedCommandText = RemoveCommentsAndStatementHint(commandText).Trim();
            commandText = commandText.Trim();
```

Reply:
Done

Reply:
Done